lib/usb.ts

```diff
@@ -37,11 +37,26 @@ export const isChromeOS105 = (): boolean => {
   return /CrOS/.test(userAgent) && /Chrome\/105\b/.test(userAgent);
 };
 
+export enum DeviceConnectMode {
```

Comment:
Let's use upper camel case values if we want a string enum here.

"Stored" feels confusing here but it's hard to improve on. Because we just mean the field, at the time you provide the option there can't be a stored one, right? Maybe we should narrow the DeviceConnectMode to just be about how it chooses a device if it doesn't have one?

I think "Known" should be "Allowed" to use terminology that matches the WebUSB API.